tutorials/ar-subs/index.md

```diff
@@ -0,0 +1,795 @@
+
+        python3 mic_identify.py
+
+The command should output something similar to this - with the example USB
+microphone being the first entry listed here:
+
+
+    (0, 'Samson Meteor Mic: USB Audio (hw:2,0)', 2)
+    (1, 'dmix', 0)
+
+2. Note down the device number - in the above example it is: `0`
+
+
+### Test recording with the USB microphone in Python
+
+Let’s try to record audio with the identified USB microphone device,
+using `pyaudio`.
+
+1. Execute the following command. The command should record a 3 second WAV
+audio file (mono, 16bits, 44.1KHz). In this example we use the device `0` as
+identified with the previous command:
+
+        python3 mic_test.py --dev 0
+
+The command should output something similar to:
+
+
+    *** Recording 3 seconds with USB device 0 ***
+    *** Finished recording. Wrote output: test1.wav ***
+
+Note - you may get ALSA errors but you can ignore them, if the recording was
+successful.
+
+2. Listen to the test recording to make sure it worked and that the dialog
+sound quality is ok. Execute:
+
+        aplay test1.wav
+
+You may need to configure which interface the Pi uses for sound playback
+output. You can choose between the HDMI output and the 3.5mm line out /
+headphone jack, by executing: `sudo raspi-config`, and configuring the setting
+under `7: Advanced Options → A4 Audio → Choose the audio output (HDMI |
+Headphones)`.
+
+
+### Test the Media Translation API example Python client
+
+Now that your USB microphone works with Python, we can try to call the Media
+Translation API. This step will test calling the API in streaming mode, piping
+microphone audio to the service and displaying the translated live responses in
+ the command line shell.
+
+
+1. Execute the utility with the following command. You can specify the [target language code](https://cloud.google.com/translate/media/docs/languages). The default target language is `de-DE` for German. For example to test with Italian, execute the below command.
+In this example we use device number `0`. Replace that with your device number if
+necessary:
+
+        python3 translate-microphone.py --lang it-IT --dev 0
+
+Note: some of the target languages require a
+[Unicode](https://home.unicode.org/) font to be displayed correctly. By default
+the Raspberry Pi console font cannot display Unicode characters. For this
+reason, use a Latin-based language such as German or Italian in this step, to
+test the Media Translation API with your microphone. The next chapters will
+show how to use the `ar-subs.py` app which uses the pygame library and
+specific fonts, to correctly display output in for example Hindi and Japanese.
+
+
+### Test Pygame with an HDMI display
+
+Now that the microphone is ready and the Media Translation API works,  make sure that pygame can control the connected HDMI display or projector. We can test this by running an example game included in the pygame package.
```

Comment:
Use 'you' instead of 'we' to address the reader, as you do in the rest of the tutorial

Reply:
Done